.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
The original regex includes a dash (`-`) which is not reported here.

I can't remember why we needed that. Just noticed the difference.
cc @jasondellaluce

Reply:
We likely wanted to support the dash to have tags for release candidates such as `v0.7.0-rc1`. Not sure if we want those tags to be released with GoReleaser though!

Reply:
Well i think we can easily add support for that in the regex, right? Let's do it now and forget about this :)

Reply:
Indeed i think we can post-pone this since we are not going to use it rn!